### PR TITLE
Add php keywords

### PR DESCRIPTION
**Pull request by zonuexe**
This list contained keywords that are not PHP reserved words today.

I propose adding the following keywords:

 * [Keywords and compile-time constants](https://www.php.net/manual/reserved.keywords.php)
 * [Type name for type declaration](https://www.php.net/manual/language.types.declarations.php)
 * Frequently mentioned classes/interfaces in code among [Predefined Classes](https://www.php.net/manual/en/reserved.classes.php)

### CODE REVIEW

**Comment by minad** [comment]
Thanks! Can you please propose the same addition to Company, even if you don't use it? It would be better to keep these in sync.

See: https://github.com/minad/cape/blob/51b9bf1276445faebfbad636d9b6a39dc57b22bc/cape.el#L79-L80

cc @dgutov

File: cape.el [hunk 1]
```diff
@@ -255,17 +255,20 @@ auto completion does not pop up too aggressively."
...
+     "abstract" "and" "array" "as" "bool" "break" "callable" "case" "catch"
+     "class" "clone" "const" "continue" "declare" "default" "die" "do" "echo"
+     "else" "elseif" "empty" "enddeclare" "endfor" "endforeach" "endif"
+     "endswitch" "endwhile" "enum" "eval" "exit" "extends" "false" "float"
+     "final" "finally" "for" "foreach" "fn" "function" "global" "goto" "if"
+     "implements" "include" "include_once" "instanceof" "insteadof" "interface"
+     "isset" "iterable" "list" "match" "namespace" "new" "null" "object" "or"
+     "print" "private" "protected" "public" "readonly" "require" "require_once"
+     "return" "self" "static" "string" "switch" "this" "throw" "trait" "true"
+     "try" "unset" "use" "var" "void" "while" "xor" "yield" "yeild from")
```
Comment (zonuexe) [inline cape.el:271]: A single `from` is not a keyword.

File: cape.el [hunk 2]
```diff
@@ -255,17 +255,20 @@ auto completion does not pop up too aggressively."
...
+    (php-mode ;; https://www.php.net/manual/reserved.php
+     "Closure" "Generator" "Exception" "Error" "Throwable"
      "__CLASS__" "__DIR__" "__FILE__" "__FUNCTION__" "__LINE__" "__METHOD__"
-     "__NAMESPACE__" "_once" "abstract" "and" "array" "as" "break" "case"
-     "catch" "cfunction" "class" "clone" "const" "continue" "declare"
-     "default" "die" "do" "echo" "else" "elseif" "empty" "enddeclare"
-     "endfor" "endforeach" "endif" "endswitch" "endwhile" "eval" "exception"
-     "exit" "extends" "final" "for" "foreach" "function" "global"
-     "goto" "if" "implements" "include" "instanceof" "interface"
-     "isset" "list" "namespace" "new" "old_function" "or" "php_user_filter"
```
Comment (zonuexe) [inline cape.el:266]: `php_user_filter` is a predefined class, but not used.

**Comment by zonuexe** [comment]
@minad 
Thank you.  I submitted same PR to https://github.com/company-mode/company-mode/pull/1311.

**Comment by minad** [comment]
Thanks!

**Pull request merged**